app-mobilephone/gammu/Manifest

```diff
@@ -1,2 +1,3 @@
 DIST gammu-1.33.0.tar.bz2 6535262 SHA256 b326223e36a334dbaed68252a9cf2d90fd898d6de4d1085dcf9c2320881aece3 SHA512 42252a36c7ad5a2945fcf23fcb1b5bb0201d27e8830d3b4a7f4c36bf00f12f5a0f7797e17098bda8d526d4ddc3b1808c0786dfae0feef9424516fa68bf2f3f1e WHIRLPOOL 27c0110da7b686bc7dd19f4f897c59d4e66aa6dbec78dec208fa8c90f70b80c1375c7b8a4b54fe40295d388de61efb11695823a503d1f85a46797c97ddf988fa
```

Comment:
More on commit messages:

* start summary line with `category/package: `,
* reference bugs in footer like `Bug: https://bugs.gentoo.org/NNNNNN` to ref or `Closes: ...` to close as fixed automatically.